platforms/xamarin.ios/System.cs

```diff
@@ -8066,51 +8269,51 @@ public partial class Socket : System.IDisposable
         public int EndReceiveFrom(System.IAsyncResult result, ref System.Net.EndPoint end_point) { throw null; }
         public int EndReceiveMessageFrom(System.IAsyncResult asyncResult, ref System.Net.Sockets.SocketFlags socketFlags, ref System.Net.EndPoint endPoint, out System.Net.Sockets.IPPacketInformation ipPacketInformation) { ipPacketInformation = default(System.Net.Sockets.IPPacketInformation); throw null; }
-        public int EndSend(System.IAsyncResult result) { throw null; }
+        public int EndSend(System.IAsyncResult asyncResult) { throw null; }
         public int EndSend(System.IAsyncResult asyncResult, out System.Net.Sockets.SocketError errorCode) { errorCode = default(System.Net.Sockets.SocketError); throw null; }
         public void EndSendFile(System.IAsyncResult asyncResult) { }
         public int EndSendTo(System.IAsyncResult result) { throw null; }
         ~Socket() { }
         public object GetSocketOption(System.Net.Sockets.SocketOptionLevel optionLevel, System.Net.Sockets.SocketOptionName optionName) { throw null; }
         public void GetSocketOption(System.Net.Sockets.SocketOptionLevel optionLevel, System.Net.Sockets.SocketOptionName optionName, byte[] optionValue) { }
-        public byte[] GetSocketOption(System.Net.Sockets.SocketOptionLevel optionLevel, System.Net.Sockets.SocketOptionName optionName, int length) { throw null; }
-        public int IOControl(int ioctl_code, byte[] in_value, byte[] out_value) { throw null; }
+        public byte[] GetSocketOption(System.Net.Sockets.SocketOptionLevel optionLevel, System.Net.Sockets.SocketOptionName optionName, int optionLength) { throw null; }
+        public int IOControl(int ioControlCode, byte[] optionInValue, byte[] optionOutValue) { throw null; }
         public int IOControl(System.Net.Sockets.IOControlCode ioControlCode, byte[] optionInValue, byte[] optionOutValue) { throw null; }
         public void Listen(int backlog) { }
-        public bool Poll(int time_us, System.Net.Sockets.SelectMode mode) { throw null; }
+        public bool Poll(int microSeconds, System.Net.Sockets.SelectMode mode) { throw null; }
         public int Receive(byte[] buffer) { throw null; }
-        public int Receive(byte[] buffer, int offset, int size, System.Net.Sockets.SocketFlags flags) { throw null; }
-        public int Receive(byte[] buffer, int offset, int size, System.Net.Sockets.SocketFlags flags, out System.Net.Sockets.SocketError error) { error = default(System.Net.Sockets.SocketError); throw null; }
-        public int Receive(byte[] buffer, int size, System.Net.Sockets.SocketFlags flags) { throw null; }
-        public int Receive(byte[] buffer, System.Net.Sockets.SocketFlags flags) { throw null; }
+        public int Receive(byte[] buffer, int offset, int size, System.Net.Sockets.SocketFlags socketFlags) { throw null; }
```

Comment:
We have our own diff tooling and we ran it against the netstandard contracts which showed these naming differences (they do show up against desktop .NET as well btw., so it's just that we didn't pay enough attention to have matching parameter names in the past).

We decided to fix those inconsistencies for at least the APIs included in netstandard 😄